chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/DeveloperController.java

```diff
@@ -75,6 +72,7 @@
     @Autowired
     public DeveloperController(final DeveloperManager developerManager,
             final CertifiedProductManager cpManager,
+            final UserManager userManager,
```

Comment:
Do we need UserManager added? It doesn't look like it's used in the rest of the diff for this file